Reason: fix build on musl
Tidy up and send upstream soon...
--- meson.build
+++ meson.build
@@ -82,8 +82,7 @@ if cc.has_header_symbol('utmpx.h', 'WTMPX_FILENAME', prefix: '#define _GNU_SOURC
 elif cc.has_header_symbol('paths.h', '_PATH_WTMPX')
   config_h.set('PATH_WTMP', '_PATH_WTMPX')
 else
-  assert(run_command('test', '-e', '/var/log/utx.log').returncode() == 0, 'Do not know which filename to watch for wtmp changes')
-  config_h.set_quoted('PATH_WTMP', '/var/log/utx.log')
+  config_h.set_quoted('PATH_WTMP', '/var/log/wtmp')
 endif
 
 # compiler flags

